File: fibonacci/fibonacci.py
#!/usr/bin/python
"""

Script to generate fibonacci series in python_2.7 and python_3

"""


def fibonacci_series():
	"""
	arguments:
	----------
		none

	returns:
	--------
		v: series of int

	description:
	------------
		function to return fibonacci series from 1 to n terms
	"""
	#first and second terms
	n1 = 0
	n2 = 1
	numbers = list()

	#getting number of terms
	n = int(input("Enter the number of terms: "))
	

	if n == 0:
		numbers.append(n1)
	elif n == 1:
		numbers.append(n1)
		numbers.append(n2)
	else:
		numbers.append(n1)
		numbers.append(n2)
		for i in range(1,n):
			numbers.append(n2+n1)
			n3 = n2+n1
			n1 = n2
			n2 = n3
	print("Fibonacci Series: " + str(numbers))

if __name__ == "__main__":
	fibonacci_series()

File: fibonacci/fibonacci.swift
func fib(_ num: Int) {
	if num == 0 {
		print("Enter a digit greater than 0")
	}
	else if num == 1{
		print("Fibonacci: 0")
	}
	else {
		var f = 0
		var f2 = 1
		var ftemp = 0
		print("Fibonacci: \(f), \(f2)", terminator:"")
		for _ in 0...num-3 {
			print(", \(f + f2)", terminator:"")
			ftemp = f + f2
			f = f2
			f2 = ftemp
		}
	}
}

print("Enter number of Fibonacci terms: ")
let input = Int(readLine()!)
fib(input!)
